src/jit/jiteh.cpp

```diff
@@ -637,6 +660,59 @@ bool            Compiler::ehIsBlockEHLast(BasicBlock* block)
            (ehIsBlockHndLast(block) != nullptr);
 }
 
+/******************************************************************************************
+* Return the EH descriptor for the most nested region that may handle exceptions raised in this BasicBlock
+* (or nullptr if this block's exceptions propagate to caller).
+*/
+EHblkDsc*       Compiler::ehGetBlockExnFlowDsc(BasicBlock* block)
+{
+  EHblkDsc* hndDesc = ehGetBlockHndDsc(block);
+
+  if ((hndDesc != nullptr) && hndDesc->InFilterRegionBBRange(block))
+  {
+    // If an exception escapes a filter, or if 'false' is returned at
+    // the end of a filter, the (original) exception is propagated to
+    // the next outer handler.  The "next outer handler" is the handler
```

Comment:
minor wording, but I would say, "if an exception is thrown in a filter, or if exception_continue_search (0) is returned at the end of the filter". "escapes" has the wrong connotations, IMO. any exception will "escape" since you can't have protected regions in a filter.